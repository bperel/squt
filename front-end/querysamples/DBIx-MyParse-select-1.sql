SELECT * FROM table1 LEFT JOIN table2 ON field1 = field2
