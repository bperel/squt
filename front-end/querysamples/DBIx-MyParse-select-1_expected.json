{
   "Tables" : {
      "table1" : {
         "table1" : {
            "OUTPUT" : {
               "*" : {
                  "-1" : "*"
               }
            },
            "CONDITION" : {
               "field1" : {
                  "JOIN" : {
                     "table2.field2" : "JOIN_TYPE_LEFT"
                  }
               }
            }
         }
      },
      "table2" : {
         "table2" : {
            "OUTPUT" : {
               "*" : {
                  "-1" : "*"
               }
            }
         }
      }
   }
}
